registry/client/errors.go

```diff
@@ -47,7 +47,7 @@ func parseHTTPErrorResponse(r io.Reader) error {
 	return errors
 }
 
-func handleErrorResponse(resp *http.Response) error {
+func HandleErrorResponse(resp *http.Response) error {
```

Comment:
Please document exported functions.

@dmcgowan Is this okay to export? Do we see any issues with that?